### ISSUE TITLE
[B站视频]谁来试试这个链接，我的好像不能下载

### ISSUE THREAD

**Issue by shanmm** [soimort/you-get#1262]
E:\Mov>you-get http://bangumi.bilibili.com/anime/v/91393
you-get: [error] oops, something went wrong.
you-get: don't panic, c'est la vie. please try the following steps:
you-get:   (1) Rule out any network problem.
you-get:   (2) Make sure you-get is up-to-date.
you-get:   (3) Check if the issue is already known, on
you-get:         https://github.com/soimort/you-get/wiki/Known-Bugs
you-get:         https://github.com/soimort/you-get/issues
you-get:   (4) Run the command with '--debug' option,
you-get:       and report this issue with the full output.

我的是这样的，经测试B站其他视频可以下载
测试网址：http://bangumi.bilibili.com/anime/v/91393

E:\Mov>you-get http://www.bilibili.com/video/av5196492/?tg
Site:       bilibili.com
Title:      【日剧日影混剪】夏天到了，现充们都在干什么？
Type:       Flash video (video/x-flv)
Size:       52.65 MiB (55208337 Bytes)

Downloading 【日剧日影混剪】夏天到了，现充们都在干什么？.flv ...
 8.5% (  4.5/ 52.7MB) ├████─────────────────────────────────────┤[1/1]    1 MB/s

**Comment by zhangn1985** [comment]
```
eid = match1(self.url, 'anime/v/(\d+)')     
self.vid = str(json.loads(get_content('http://bangumi.bilibili.com/web_api/episode/get_source?episode_id={}'.format(eid)))['result']['cid'])
```

**Comment by e788b1** [comment]
```bash
you-get --debug http://bangumi.bilibili.com/anime/v/91393
[DEBUG] get_content: http://bangumi.bilibili.com/anime/v/91393
[DEBUG] get_content: http://www.bilibili.com/video/av5224238/
[DEBUG] get_content: http://interface.bilibili.com/playurl?appkey=f3bb208b3d081dc8&cid=8490017
Site:       bilibili.com
Title:      【7月】霹雳布袋戏 东离剑游纪 特番【独家正版】
Type:       Flash video (video/x-flv)
Size:       341.85 MiB (358453821 Bytes)

Downloading 【7月】霹雳布袋戏 东离剑游纪 特番【独家正版】.flv ...
 3.9% ( 13.2/341.8MB) ├███─────────────────────────────────────────────────────────────────────────────┤[1/4]   11 MB/s
```

**Comment by shanmm** [comment]
@e788b1 E:\Mov>you-get --debug http://bangumi.bilibili.com/anime/v/91393
[DEBUG] get_content: http://bangumi.bilibili.com/anime/v/91393
you-get: version 0.4.455, a tiny downloader that scrapes the web.
you-get: ['http://bangumi.bilibili.com/anime/v/91393']
Traceback (most recent call last):
  File "<string>", line 11, in <module>
  File "src\you_get\__main__.py", line 92, in main
  File "src\you_get\common.py", line 1291, in main
  File "src\you_get\common.py", line 1204, in script_main
  File "src\you_get\common.py", line 1041, in download_main
  File "src\you_get\common.py", line 1284, in any_download
  File "src\you_get\extractors\bilibili.py", line 131, in bilibili_download
AssertionError
you-get returned -1
为什么我的是这样的

**Comment by shanmm** [comment]
@zhangn1985 看不到那个代码

**Comment by comwrg** [comment]
```
C:\Users\Wrg\Desktop\you-get>python you-get http://bangumi.bilibili.com/anime/v/
91393
Site:       bilibili.com
Title:      【7月】霹雳布袋戏 东离剑游纪 特番【独家正版】
Type:       Flash video (video/x-flv)
Size:       341.85 MiB (358453821 Bytes)

Downloading 【7月】霹雳布袋戏 东离剑游纪 特番【独家正版】.flv ...
 0.0% (  0.0/341.8MB) ├────────────────────────────
 0.1% (  0.2/341.8MB) ├────────────────────────────
 0.1% (  0.5/341.8MB) ├────────────────────────────
 0.2% (  0.8/341.8MB) ├────────────────────────────
 0.3% (  1.0/341.8MB) ├────────────────────────────
 0.4% (  1.2/341.8MB) ├────────────────────────────
 0.4% (  1.5/341.8MB) ├────────────────────────────
 0.5% (  1.8/341.8MB) ├────────────────────────────
 0.6% (  2.0/341.8MB) ├────────────────────────────
 0.7% (  2.2/341.8MB) ├────────────────────────────
 0.7% (  2.5/341.8MB) ├────────────────────────────
 0.8% (  2.8/341.8MB) ├────────────────────────────
─────────────┤[1/4]    5 MB/s
```


测试没问题

**Comment by shanmm** [comment]
@comwrg 看來只有我才有問題

**Comment by comwrg** [comment]
@shanmm 重新下载下看看行不

**Comment by shanmm** [comment]
@comwrg 不行，今天試了好多次了都，哎可能是我電腦的問題

**Comment by soimort** [comment]
@soimort-bot Test this with stable: http://bangumi.bilibili.com/anime/v/91393

**Comment by soimort** [comment]
@soimort-bot Test this again with `develop`: http://bangumi.bilibili.com/anime/v/91393

**Comment by soimort** [comment]
Fixed by d8aca8f5427d4e365da817288906cff9dd88bade.

需要升级到git的最新版本。
